sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
```

Comment:
Why are we removing this space?  If I remember correctly, there are two slightly different entries for the vti interface in the list returned by SNMP, and we want the one preceded by a space.